project/accounts/templates/accounts/account.html

```diff
@@ -32,13 +31,21 @@
           </a>
       {% endif %}
     {% endif %}
-
     <div class="user-info section">
       <div class="full-name">
         {{ profile.first_name }} {{ profile.last_name }}
       </div>
       <div class="username">@{{ username }}</div>
     </div>
+    <div class="categories">
+      <ul>
+        {%for categories in profile.categories.all%}
+        <li>
+          {{categories}}
```

Comment:
Let's use the singular word `category` since each item represents a single profile category.

```suggestion
        {%for category in profile.categories.all%}
        <li>
          {{ category }}
```

Reply:
Okay, perfect. Let me make the necessary changes. Thanks.